library/src/main/res/values/attrs.xml

```diff
@@ -171,6 +171,7 @@
     <!--MessageListView-->
     <declare-styleable name="MessageListView">
         <!--Scroll button-->
+        <attr name="streamDefaultButtonEnabled" format="boolean" />
         <attr name="streamButtonBackground" format="reference" />
         <attr name="streamButtonIcon" format="reference" />
```

Comment:
Could we give these attributes a more specific name than "Button"? Without looking at the comment above them, it's hard to tell what these do, and that's not there when clients are setting attributes on `MessageListView` in their own layout files.

Reply:
ScrollButton perhaps?

Reply:
Yup, that sounds descriptive enough 👍 . If we change it, let's make sure it's in the changelog as well, because these attributes might already be used by some clients (though they're pretty new 😄 ).